x/clp/keeper/calculations_test.go

```diff
@@ -886,8 +886,8 @@ func TestKeeper_GetSwapFee(t *testing.T) {
 	nativeAssetAmount := sdk.NewUintFromString("998")
 	externalAssetAmount := sdk.NewUintFromString("998")
 	asset := types.NewAsset("eth")
-	externalCoin := sdk.NewCoin(asset.Symbol, sdk.Int(sdk.NewUint(10000)))
-	nativeCoin := sdk.NewCoin(types.NativeSymbol, sdk.Int(sdk.NewUint(10000)))
+	externalCoin := sdk.NewCoin(asset.Symbol, sdk.NewIntFromUint64(10000))
+	nativeCoin := sdk.NewCoin(types.NativeSymbol, sdk.NewIntFromUint64(10000))
 	err := sifapp.AddCoinsToAccount(types.ModuleName, app.BankKeeper, ctx, signer, sdk.NewCoins(externalCoin, nativeCoin))
```

Comment:
Probably best to add these changes in a separate PR since they're not related to the ante handler. You could even PR these against the feature/clp-refactor branch.